net.newpipe.NewPipe.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop-application">
+  <id>net.newpipe.NewPipe</id>
+  <name>NewPipe (experimental Flatpak version)</name>
+  <summary>A free lightweight YouTube frontend for Android.</summary>
+  <project_license>GPL-3.0-or-later</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="vcs-browser">https://github.com/TeamNewPipe/NewPipe</url>
+  <url type="homepage">https://newpipe.net/</url>
+  <url type="bugtracker">https://gitlab.com/android_translation_layer/android_translation_layer/-/issues</url>
+  <url type="donation">https://newpipe.net/donate/</url>
+  <url type="translate">https://hosted.weblate.org/engage/newpipe/</url>
+  <description>
+    <p>NewPipe does not use any Google framework libraries, or the YouTube API. It only parses the website in order to gain the information it needs. Therefore this app can be used on devices without Google Services installed. Also, you don't need a YouTube account to use NewPipe, and it's FLOSS.</p>
+    <p>This is an unofficial Flatpak build based on Android Translation Layer (<a href="https://gitlab.com/android_translation_layer/android_translation_layer">https://gitlab.com/android_translation_layer/android_translation_layer</a>). Please report bugs to the ATL bug tracker instead of the NewPipe bug tracker.</p>
```

Comment:
```suggestion
    <p>This is an unofficial Flatpak build based on Android Translation Layer. Please report bugs to the ATL bug tracker instead of the NewPipe bug tracker https://gitlab.com/android_translation_layer/android_translation_layer/-/issues.</p>
```

Reply:
It's not HTML.

Reply:
Also make this the first `<p>` tag and add `<p><em>NOTE:</em> This is an unofficial...</p>`

Reply:
done